### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by themanforfree**

### CODE REVIEW

File: xline/src/main.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //! Xline
 #![deny(
-    // The following are allowed by default lints according to
+    // The following are allowed by default &lints according to
```
Comment (pzheng1025) [inline xline/src/main.rs:3]: remove &

File: xline/src/main.rs [hunk 2]
```diff
@@ -110,19 +110,12 @@
     clippy::multiple_crate_versions, // caused by the dependency, can't be fixed
 )]
 
-use std::net::{AddrParseError, SocketAddr};
-
 use anyhow::Result;
 use clap::Parser;
 use log::debug;
+use std::net::SocketAddr;
```
Comment (pzheng1025) [inline xline/src/main.rs:116]: std need to be put before third party libs

Comment (themanforfree) [reply xline/src/main.rs]: cargo fmt will sort them alphabetically,  this modification will make format CI fail

Comment (pzheng1025) [reply xline/src/main.rs]: You need to add a empty line after std, please check other files

**Pull request closed without merging**